ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx

```diff
@@ -142,6 +142,11 @@ const WorkflowNodeSummary = (props: Props) => {
                         <i className='fa fa-bars' /> main logs
                     </DropDownButton>
                 )}{' '}
+                {props.node.type === 'Pod' && props.onShowEvents && (
```

Comment:
Moved events next to logs:

![image](https://user-images.githubusercontent.com/1142830/105527518-a5b7c280-5c98-11eb-940d-88805820ecba.png)